### ISSUE TITLE
go get doesn't work due to github.com/influx6/{moz,faux} dependencies

### ISSUE THREAD

**Issue by bzub** [gu-io/gu#38]
The vendored influx6 repos seem to have breaking changes included within them. I was trying to sort it out in cmd/gu/main.go, but I'm just starting with Gu. Very interested in using this project, thanks in advanced for your help.

```
go get -u -v github.com/gu-io/gu/...                                                                                                                                                                                                        
github.com/gu-io/gu (download)                                                                                                                                                                                                                 
github.com/influx6/moz (download)                                                                                                                                                                                                              
package github.com/influx6/moz/annotations: cannot find package "github.com/influx6/moz/annotations" in any of:                                                                                                                                
        /usr/lib/go/src/github.com/influx6/moz/annotations (from $GOROOT)                                                                                                                                                                      
        /home/bzub/go/src/github.com/influx6/moz/annotations (from $GOPATH)                                                                                                                                                                    
github.com/influx6/faux (download)                                                                                                                                                                                                             
package github.com/influx6/faux/utils: cannot find package "github.com/influx6/faux/utils" in any of:                                                                                                                                          
        /usr/lib/go/src/github.com/influx6/faux/utils (from $GOROOT)                                                                                                                                                                           
        /home/bzub/go/src/github.com/influx6/faux/utils (from $GOPATH)                                                                                                                                                                         
github.com/gopherjs/gopherjs (download)
```

**Comment by bzub** [comment]
I'm currently trying to figure out how to migrate gu code after this change in moz: https://github.com/influx6/moz/commit/7b3d5bb40884abd567ea183747cd7610d405187a

**Comment by influx6** [comment]
I see thanks for the shout will get to eat immediately tomorrow and get this sorted.

**Comment by influx6** [comment]
Issues should be fixed in  559d04d commit.
